### DISCUSSION TITLE
Why does this happen when I keep sync audio and midi to always?

### DISCUSSION THREAD

**Discussion by JamesXXV** [apohl79/audiogridder#836] [Help]
With single instance of audiogridder active on a drum kit:

<img width="277" alt="Screen Shot 2021-12-05 at 3 12 20 AM" src="https://user-images.githubusercontent.com/94138763/144725490-b154d872-d09b-463d-8aff-d57abc44b654.png">

With no instances of audiogridder:

<img width="276" alt="Screen Shot 2021-12-05 at 3 13 01 AM" src="https://user-images.githubusercontent.com/94138763/144725505-c31fe811-8dad-4381-b4c8-d81d3cb351b0.png">

This happens on both my M1 Mac and my intel Mac the live connection basically hogs a single core leading to crackling a lot. and it worsens on low buffer settings. but the same doesn't happen when I use Vienna Ensemble Pro so maybe this is fixable? don't get me wrong this is a really great plugin but this crackling makes it almost not usable during production. mixing is fine I can crank up the buffer to 1024 samples and be done with it but in low latency environments even a single instance can crash coreaudio. So what I am trying to say is is there something I can do to use it in low latency say 256 samples buffer size?

Important edit: I switched on the buffer in the audio gridder plugin itself and I was able to help some of the crashing tho the chain adds around 85 ms on 2 blocks and the +1block setting doesn't really help much in terms of processor load so I had to use the 2 blocks, so I guess I'll have to live with it? Also is the 2 block latency per plugin or the whole chain say I have an audiogridder instance on my audio track and my master channel will I be looking at a latency of 170 ms or just the 85 ms selected.

**Comment by Henri75** [comment]
I saw too some weird issues like yours in Cubase, but it is really better with beta 6 (but still there) with low buffer.

I asked myself the same question about buffer and latency, Andreas will confirm but it seems it is global (so for all AG instances in a session the buffer is shared)
